extension/options.html

```diff
@@ -13,16 +17,23 @@
       margin: 0;
       padding: 0;
     }
-    li {
-      display: grid;
-      grid-template-columns: auto 100px 1fr 16px;
-      column-gap: 4px;
-    }
     </style>
     <script type="module" src="options.js"></script>
   </head>
   <body>
+    <h1>Preferences</h1>
+    <section id="preferences">
+      <ul>
+        <li>
+          <input id="css-value-enabled" type="checkbox">
+          <label for="css-value-enabled">
+            Enable to analyse CSS value issues (This is very experimental)
```

Comment:
Please reword to "Show warnings for CSS values (Experimental)"

Reply:
Okay, thanks!